mongodbatlas/resource_mongodbatlas_backup_compliance_policy.go

```diff
@@ -39,6 +39,16 @@ func resourceMongoDBAtlasBackupCompliancePolicy() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"authorized_user_first_name": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"authorized_user_last_name": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
```

Comment:
is the API returning these fields even when they are not set? if the answer is not, we can remove the compute here

Reply:
yes they are being returned as `unknown` hence that's why I used `Computed: true`.

Reply:
Thanks for the clarification here